ports/async-mqtt/portfile.cmake

```
@@ -0,0 +1,27 @@
+vcpkg_from_github(
+    OUT_SOURCE_PATH SOURCE_PATH
+    REPO redboltz/async_mqtt
+    REF 1.0.1
```

```suggestion
    REF "${VERSION}"
```
We can use the new function.